Berlin/BerOr1270.xml

```diff
@@ -24,16 +30,265 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
             <msDesc xml:id="ms">
                <msIdentifier>
                   <repository ref="INS0333SBB"/>
-                  <collection>Ms. orient. oct.</collection>
+                  <collection>Ethiopic</collection>
                   <idno>SBP, Ms. orient. oct. 1270</idno>
                </msIdentifier>
                <msContents>
+                 <summary></summary>
 <msItem xml:id="ms_i1">
-  <locus from="2ra" to="24va"/>
-<title ref="LIT1450GadlaG">Gadla Gabra Krǝstos</title>
+  <locus from="2ra" to="16vb"/>
+  <locus from="19ra" to="24va"/>
+  <note>The folios of the manuscript are misplaced, being rearranged wrongly together: the text is in disorder.</note>
+  <title type="complete" ref="LIT1450GadlaG"/>
+  <textLang mainLang="gez"/>
+  <incipit xml:lang="gez">
+  <locus target="#2ra"></locus>
+  በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ አሐዱ፡ አምላክ፨ ንቀድም፡ እንከ፡ በረድኤተ፡ እግዚእነ፡ ኢየሱስ፡ ክርስቶስ፨ ንጽሕፍ፡ ዜናሁ፡ ለቅዱስ፡ ብእሴ፡ እግዚኣብሔር፡ <persName ref="PRS1661Alexis">ገብረ፡ ክርስቶስ</persName>።
+  ወልደ፡ <persName ref="PRS">ቴዎዶስዮስ፡</persName> ንጉሥ፡ ዘ<placeName ref="LOC2196Consta">ቈስጠንጥንያ</placeName>፨፨
+</incipit>
 </msItem>
-               </msContents>
-            </msDesc>
+<msItem xml:id="ms_i2">
+  <locus from="17ra" to="18vb"/>
+  <locus from="25ra" to="25vb"/>
+  <locus from="28ra" to="38va"/>
+  <note>Number of Miracles 8. The folios are misplaced, the text is in disorder. The right order of the miracles is: ff. 28ra-35vb, 36ra-36vb, 17ra-18vb, 37ra-37vb, 25ra-25vb, 38ra-38va. Folios missing between f. 35 and f. 36, with consequent missing part of the text.</note>
+  <title type="incomplete" ref="LIT2384Taamme"/>
+  <textLang mainLang="gez"/>
+<incipit xml:lang="gez">
+<locus target="#28ra"></locus>
+በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ አሐዱ፡ አምላክ። ንቀድም፡ እንከ፡ በረዴተ፡ እግዚእነ፡ ኢየሱስ፡ ክርስቶስ፨ ወእምዝ፡ ንጽሕፍ፡ ተአምሪሃ፡ ለእግዝእትነ፡ ማርያም፡ ወላዲተ፡ አምላክ፡
+</incipit>
+<msItem xml:id="ms_i2.1">
+  <locus from="28rb" to="30ra"></locus>
+    <title type="complete"  subtype="miracle 1"></title>
```

Comment:
Here, using the attribute subtype is not correct. The confusion might stem from the fact that it is used in <div type="edition"> in ms and work records. Instead of subtype, use ref and then insert the correct work ID (We should have IDs for most miracles of Mary, you can check the PEMM incipit tool).

Reply:
You are right, Dorothea. I will use the PEMM incipit tool, then.

Reply:
Miracle 1 has e.g. the ID 106; If I write only ref="106", it gives me a mistake. It is missing something, but I don't know understand what, sorry!

Reply:
Where shall I look in the BM database, to find the list of Miracles with ID?

Reply:
Ah, you are right, in my memory the CAe was present in the PEMM tool, but it's not. If the miracle is one that has a Macomber ID in the PEMM tool, you can check the list of cited publications https://betamasaheft.eu/bibliography, write bm:MacomberMiracles in the "enter a zotero ID" field and then search among the results for "item, 106".
Another way is to search in BM with the incipit, title translation or some key words. In this example, I searched "beggar" and quickly found  LIT3640Miracle. This is the ID that should be used in ref.

Reply:
Dear Dorothea, yes, I opened the  https://betamasaheft.eu/bibliography, and found it. Great! Thank you very much for the clarification. Yes, Right! The PEMM incipit tool has only the ID, but not CAe..